interview-questions.md

```diff
@@ -14,44 +14,45 @@ Researched answer:
 
 1. What is git? What is the difference between git and Github?
 
-Your answer:
+Your answer: Git is a version control software used on local computer. GitHub is version control software cloud based.
 
-Researched answer:
+Researched answer: Git is a version control system that allows you to track source code history. GitHub is cloud based service that allows you to manage repositories and keep track of them. 
 
 2. Which JavaScript operators will return a Boolean value?
 
-Your answer:
+Your answer: Logical operators
```

Comment:
When answering interview questions, you should expand using definitions and examples when possible. One-word or two-word answers in an interview setting will not show your understanding and knowledge, and will likely come off as uninterested.